diff --git i/contanki/funcs.py w/contanki/funcs.py
index ad54c72..7fb0a2c 100644
--- i/contanki/funcs.py
+++ w/contanki/funcs.py
@@ -72,7 +72,7 @@ def get_config() -> dict:
     else:
         config.update(loaded_config)
 
-    mw.addonManager.writeConfig(__name__, config)
+    # mw.addonManager.writeConfig(__name__, config)
     return config
 
 
diff --git i/contanki/utils.py w/contanki/utils.py
index bdfd119..3c987ba 100644
--- i/contanki/utils.py
+++ w/contanki/utils.py
@@ -11,17 +11,37 @@ import re
 from typing import Literal
 from os.path import join, dirname, abspath, exists
 from os import environ
+import os
+import shutil
+import stat
 
 DEBUG = environ.get("DEBUG")
 
 addon_path = dirname(abspath(__file__))
 tests_path = join(addon_path, "tests")
-user_files_path = join(addon_path, "user_files")
+user_files_path = os.path.expanduser("~/.local/share/Anki2/addons21/1898790263/user_files")
 user_profile_path = join(user_files_path, "profiles")
 user_controllers_path = join(user_files_path, "custom_controllers")
 default_profile_path = join(addon_path, "profiles")
 controllers_path = join(addon_path, "controllers")
 
+os.makedirs(user_files_path, exist_ok=True)
+for item in os.listdir(join(addon_path, "user_files")):
+    src = join(addon_path, "user_files", item)
+    dst = join(user_files_path, item)
+    if not exists(dst):
+        if os.path.isdir(src):
+            shutil.copytree(src, dst)
+        else:
+            shutil.copy2(src, dst)
+
+        os.chmod(dst, os.stat(dst).st_mode | stat.S_IWUSR)
+        if os.path.isdir(dst):
+            for root, dirs, files in os.walk(dst):
+                for name in dirs + files:
+                    path = join(root, name)
+                    os.chmod(path, os.stat(path).st_mode | stat.S_IWUSR)
+
 State = Literal[
     "all",
     "deckBrowser",
