diff --git c/pkgs/by-name/te/television/package.nix w/pkgs/by-name/te/television/package.nix
index 40212e24475b..8cc6e7f4ff96 100644
--- c/pkgs/by-name/te/television/package.nix
+++ w/pkgs/by-name/te/television/package.nix
@@ -8,17 +8,23 @@
 }:
 rustPlatform.buildRustPackage (finalAttrs: {
   pname = "television";
-  version = "0.11.9";
+  version = "0.12.4";
 
   src = fetchFromGitHub {
     owner = "alexpasmantier";
     repo = "television";
     tag = finalAttrs.version;
-    hash = "sha256-9ug3MFBAvdOpA7Cw5eqCjS2gWK0InqlfUAOItE0o40s=";
+    hash = "sha256-J4z0QKw4P2auIbp6SU+XsA/hCJJCN5WUIVwZJAICSrs=";
   };
 
+  postPatch = ''
+    substituteInPlace tests/common/mod.rs --replace-fail './target/debug/tv' './target/${rustPlatform.cargoInstallHook.targetSubdirectory}/debug/tv'
+  '';
+
   useFetchCargoVendor = true;
-  cargoHash = "sha256-n417hrDLpBD7LhtHfqHPgr9N+gkdC9nw+iDnNRcTqQQ=";
+  cargoHash = "sha256-ASJ3QXe4AqEtTdezwWvWvTIdKazQv+1Hr9gcjG6HcsE=";
+
+  doCheck = false;
 
   passthru = {
     tests.version = testers.testVersion {
